content/about/principles/planning.md

```diff
@@ -0,0 +1,58 @@
+---
+linkTitle: "Planning"
+title: "ReproNim Principle 1: Study planning"
+type: docs
+weight: 80
+---
+
+Study planning represents a foundational step in ensuring reproducible neuroimaging research.  Many problems that lead to reproducibility issues can be avoided by proper planning 
+
+1. **Implement good science basics, e.g, power analysis, statistical consults**  
+   If your study is underpowered or your study design is flawed, you will have trouble publishing your work and, more to the point, your work will likely not be reproducible ([Scuzs and Ioannidis, 2020](https://www.sciencedirect.com/science/article/pii/S1053811920306509#bib24)).   
+   * ***ReproNim resources*****:**   
+     * [Repronim training module in statistics for neuroimaging](https://www.repronim.org/module-stats/):  covers all aspects of statistics for neuroimaging, including sampling, power analysis and more
+
+   * ***Other resources:***    
+     * NINDS-funded [Community for Rigor (C4R)](https://c4r.io/): tools and resources for improving rigor and reproducibility, from basics to advanced
+
+     * Statistical consult: Most universities have statisticians available for consultation. The most important time to consult with a statistician is at the conceptualization phase, ***not*** after the data are collected.
+
+2. **Use pre-existing data for planning and/or analysis**  
+   Leveraging pre-existing data, whether your own or data available from public repositories,  can help validate methodological approaches before collecting new data and potentially increase your sample size
+
+   * ***ReproNim resources*****:**   
```

Comment:
we need to have a look at the deploy preview and fixup this and other odd formatting